src/io/flutter/run/common/CommonTestConfigUtils.java

```diff
@@ -160,7 +161,7 @@ protected ActiveEditorsOutlineService getActiveEditorsOutlineService(@NotNull Pr
    * Traverses the {@param outline} tree and adds to {@param callToTestType } the {@link DartCallExpression}s that are tests or test groups.
    */
   private void populateTestTypeMap(@NotNull FlutterOutline outline,
-                                   @NotNull Map<DartCallExpression, TestType> callToTestType,
+                                   @NotNull Map<Integer, TestType> callToTestType,
                                    @NotNull PsiFile file) {
     final PsiElement element = file.findElementAt(outline.getOffset());
```

Comment:
optimization: move this findElementAt call so you only call it when unit_test_group or unit_test_test is found. Currently we are doing this fairly expensive operation a lot more times than we need to.
I do like that you are still using
file.findElementAt to avoid location calculation bugs.

Reply:
Done.